Formula/openshift-cli.rb

```diff
@@ -1,56 +1,20 @@
 class OpenshiftCli < Formula
-  desc "OpenShift command-line interface tools"
+  desc "Red Hat OpenShift command-line interface tool"
   homepage "https://www.openshift.com/"
-  url "https://github.com/openshift/oc.git",
-      tag:      "openshift-clients-4.6.0-202006250705.p0",
-      revision: "51011e4849252c723b520643d27d3fa164d28c61",
-      shallow:  false
-  version "4.6.0"
+  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.7.2/openshift-client-mac-4.7.2.tar.gz"
```

Comment:
This ships a platform-specific pre-built binary, which is not acceptable in homebrew/core.

https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

This _may_ be acceptable at homebrew/cask, which has no such requirement.

Reply:
Alright, I'll use the correct formatting next time. I'm also closing this and I'll figure out the new tagging style and setup proper bottles.